src/themes/default/spacing.ts

```diff
@@ -8,6 +8,6 @@ export const spacing = {
   extraLarge: 32,
```

Comment:
The nesting here with `src/themes/default` might accidentally imply we'd have other folders inside the `themes` folder within the SDK.  But we don't intend on doing that atm, right?  I don't have a better suggestion and dig this for now, just noting the potential confusion.

Reply:
Correct, maybe better to just use `src/defaultTheme/`... However, thinking about now, do we need this at the root of `src`. Since it isn't meant to imported directly, but instead accessed through `useBrandConfig` then maybe we just put it with the other branding files.. next to the `Theme` class which is it's only direct consumer.

Reply:
Oh, so it wouldn't have a folder name at all - just `colors.ts` and `spacing.ts` files as siblings to `Theme` - love it.  We could prefix like `baseColors.ts` too - I personally like that better but either way love the idea of removing the folder.